src/UpdateTracker.js

```diff
@@ -57,11 +60,17 @@ function trackResource(url) {
   if (!webSocket) {
     const socketUrl = `${protocol.replace('http', 'ws')}//${host}/`;
     webSockets[host] = webSocket = new WebSocket(socketUrl);
+    const backOffDelay = delay || 1000;
     Object.assign(webSocket, { enqueue, onmessage,
       ready: new Promise(resolve => (webSocket.onopen = resolve)),
+      onclose: oncloseFor(host),
+      resources: [],
     });
+    setUpBackOff(webSocket, retryAttempt || 0, backOffDelay);
   }
 
+  webSocket.resources.push(url); // Storing URL of resouce that will be subscribed to
```

Comment:
Typo; can we put the comment above the line, and perhaps clarify that we keep track of resources so we can resubscribe if needed?

Also, might want to make `webSocket.resources` a `Set` to avoid multiple subscriptions.

Reply:
Done.